src/renderer/state.ts

```diff
@@ -610,6 +613,12 @@ export class AppState {
       return;
     }
 
+    // Download the version without setting it as the current version.
+    if (opts?.noActivate) {
+      await this.installer.ensureDownloaded(version);
```

Comment:
This skips the options passed to `installer.install`, as well as the log statement on line 613/622. Suggestion: pull out the options into `const options = {...};` and then do `this.installer.ensureDownloaded(version, options);`.

Unrelated to this change, but it looks like there's another bug around showing progress on the spinner when downloading under Settings, it works properly on the version select. 🤔

Reply:
What do you mean here precisely? what options would apply to `ensureDownloaded` in this context?

Reply:
[These options](https://github.com/electron/fiddle/blob/4beb8aba7deb2dc4f60fe874912fe3db45251765/src/renderer/state.ts#L615-L627). Both code paths should probably be consistent in which mirrors they use, and the `progressCallback` option is definitely needed of #1243 will no longer work.

Reply:
@dsanders11 done!